File: backend/src/main/java/com/back/domain/book/client/aladin/dto/AladinBookDto.java
package com.back.domain.book.client.aladin.dto;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Builder
public class AladinBookDto {
    private String title;
    private String imageUrl;
    private String publisher;
    private String isbn13;
    private int totalPage;
    private LocalDateTime publishedDate;
    private String categoryName;
    private String mallType;
    private List<String> authors;
}

File: backend/src/main/java/com/back/domain/book/client/aladin/dto/AladinBookDto.kt
package com.back.domain.book.client.aladin.dto

import java.time.LocalDateTime

data class AladinBookDto(
    val title: String? = null,
    val imageUrl: String? = null,
    val publisher: String? = null,
    val isbn13: String? = null,
    val totalPage: Int = 0,
    val publishedDate: LocalDateTime? = null,
    val categoryName: String? = null,
    val mallType: String? = null,
    val authors: List<String> = emptyList()
)